woocommerce/wc-admin
# Fixed error hiding core task list

Fixes #6049

This PR fixes the error with hiding the core task list. After adding an extended task list, it's not possible to hide the core task list.

_Related also to [issue 6023](https://github.com/woocommerce/woocommerce-admin/issues/6023)_

### Screenshots
![Screen Capture on 2021-01-12 at 15-56-42](https://user-images.githubusercontent.com/1314156/104359766-d7899600-54ee-11eb-8140-25de1eca3ee5.gif)

### Detailed test instructions:

1. Add [a task list item](https://github.com/woocommerce/woocommerce-admin/tree/main/docs/examples/extensions/add-task). You can do it by executing something like: `npm run example -- --ext=add-task` in your terminal
2. Go to the `Home` screen.
3. Hide the core task list.
4. The task list should disappear.

- Also, go to `/wp-admin/admin.php?page=wc-admin&task=payments` and verify the _setup additional payment providers_ card is still visible.
![screenshot-one wordpress test-2021 01 12-16_05_51](https://user-images.githubusercontent.com/1314156/104360620-25eb6480-54f0-11eb-8e2c-2857b1b5a742.png)
